app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,47 @@
+class QuestionsController < ApplicationController
+
+  before_action :find_test, only: :index
+  before_action :find_question, only: [:show, :destroy]
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_not_found
+
+  def index
+    result = @test.questions.all.pluck(:body)
+    render plain: result
+  end
+
+  def show
+    render plain: @question.body
+  end
+
+  def destroy
+    Question.destroy(@question)
```

Comment:
```suggestion
    @question.destroy
```
объект ты находишь с помощью коллбека
здесь его нужно просто "дестройнуть" и вызывается этот "дестрой" как я выше написал

твой вариант не рабочий, он должен вернуть ошибку
можно использовать вариант такой:
```ruby
    Question.destroy(@question.id)
```
т.е. передавть ай-ди, а не объект
но я не помню, чтобы встречал в проектах такой код, обычно пишут в объектном стиле, как я выше предложил